System/Posix/Process/Internals.hs

```diff
@@ -9,7 +10,12 @@ import Foreign
 import Foreign.C
 import System.Exit
 import System.IO.Error
+
+#ifdef ghcjs_HOST_OS
```

Comment:
This seems the wrong place to kludge this around; why can't ghcjs' `GHC.Conc (Signal)` be used/fixed?